index.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const villains = require('./villains')
+const {getVillains} = require('./controller/villains.js')
+const app = express()
+app.get('/', getVillains)
```

Comment:
you made your slug correct but there is no call to access it.   app.get('/villains/:slug', slugger) alsoou only imported your getVillains function you need to import all the functions and give them routes and call that function with its respective route const { getVillains, slugger, addNewTeam } = require('./controller/villains.js')  otherwise there is no where locally calling for that information and if it is it has no where to actually go